interfaces/Portal/staticwebapp.config.json

```diff
@@ -42,6 +42,7 @@
     }
   },
   "globalHeaders": {
+    "Permissions-Policy": "interest-cohort=()",
```

Comment:
What is this?

Reply:
See https://cleanuptheweb.org for a nice and short, shouty explanation. :)